### PR TITLE
Small fixes to fix the site build

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

File: exist-parent/pom.xml [hunk 1]
```diff
@@ -682,7 +682,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-scm-plugin</artifactId>
-                    <version>1.11.12</version>
+                    <version>1.11.2</version>
```
Comment (dizzzz) [inline exist-parent/pom.xml]: looks like a downgrade?

Comment (adamretter) [reply exist-parent/pom.xml]: It was a typo, there is no such version as `1.11.12`, latest available version is `1.11.2`

**Comment by dizzzz** [comment]
hmmm the windows build keep failing:

https://ci.appveyor.com/project/AdamRetter/exist/builds/29397857/job/e44k1huqif5jvwif

I hope this is unrelated to the change.

**Comment by duncdrum** [comment]
@dizzzz yes windows is broken on develop, see #3153 and #3149

**Pull request merged**